docs/docs/creating-service-bindings/creating-service-binding.md

```diff
@@ -0,0 +1,71 @@
+---
+sidebar_position: 1
+---
+
+# Creating Service Binding
+
+In this section, we consider that the service you want to connect to is already
+exposing binding metadata by either:
+
+1. Provisioned Service
+2. Direct Secret Reference
+3. Secret Generated  through CRD/CR annotations
+4. Secret Generated through OLM Descriptors
+5. Detect Bindings through owned resources
+
+You also required the application ready before creating the ServiceBinding
+resource.
+
+There are two APIs supported by the Service Binding Operator.  Both has
```

Comment:
It would be nice if documentation could expand on what are the main difference between those APIs and which one is recommended.

Reply:
Added a comparison. 481088d985e0b477230270d6234efa8d855e43c5